processor/tests/src/main/java/org/mobilitydata/gtfsvalidator/processor/tests/MixedCaseSchema.java

```diff
@@ -0,0 +1,12 @@
+package org.mobilitydata.gtfsvalidator.processor.tests;
+
+import org.mobilitydata.gtfsvalidator.annotation.GtfsTable;
+import org.mobilitydata.gtfsvalidator.annotation.MixedCase;
+import org.mobilitydata.gtfsvalidator.annotation.Required;
+
+@GtfsTable("mixed_case.txt")
+public interface MixedCaseSchema {
+  @Required
+  @MixedCase
+  String mixedCase();
```

Comment:
I would actually name this field something different.  Like "someField".  I think having the field named "mixedCase" causes just a bit of confusion in the unit-test about when "mixedCase" is being used as field name or in some other context.

Reply:
I also found it a little confusing that "MixedCase" is both used as the annotation and the entity class names in the tests below.

Reply:
@bdferris-v2 I modeled the `CurrencyAmount` example, so it has the same issue. I renamed it to `MixedCaseTestSchema`, though not sure it's a major improvement in readability. Let me know what you think.

Reply:
Ha, it's true.  Maybe it was less noticeable in that test because I inlined all the file and field names in the test, as opposed to referencing CurrencyAmount.FILENAME, etc?  Sorry for the conflicting advice.